storybookjs/storybook
# Move 'others' to the top of the tree

## What I did

Most storybooks don't use roots, but when you embed/import a 3rd party storybook, that one gets a root. The result is that your own stories are no longer at the top, but buried under the imported stories and slotted under an "others" category. Besides the fact this label might not always make sense (and it's not translatable either), these stories are often the most important ones and therefore should be at the top of the tree. This has the added benefit of being able to drop the "others" label because it's not unusual to have ungrouped items at the top of such a list.

## How to test

See the [Sidebar stories](https://www.chromaticqa.com/build?appId=5a375b97f4b14f0020b0cda3&number=12917).
